examples/adder/Cargo.toml

```diff
@@ -17,7 +17,7 @@ workspaces = "0.5.0"
 serde_json = "1.0"
 tokio = { version = "1.14", features = ["full"] }
 anyhow = "1.0"
-near-abi = "0.1.0-pre.0"
+near-abi = { version = "0.3.0", features = ["__chunked-entries"] }
```

Comment:
This feature isn't part of the public API, no example should use it directly. Doesn't look like it's needed here anyway, compiles fine without it.

Reply:
oh, yeah, added by accident